draft-ietf-gnap-core-protocol.md

```diff
@@ -851,18 +851,17 @@ identifier (string)
     For example, a patient identifier for a medical API or
     a bank account number for a financial API.
 
-The following non-normative example shows the use of both common
-and API-specific fields as part of two different access `type` 
-values. 
+The following non-normative example is asking for two kinds of access to each of
+two different locations and two different data types for a single access token 
+using the fictitious `photo-api` type definition.
```

Comment:
I think we should avoid any possible interpretation issue. 

Suggested change: 

The following non-normative example is asking for two kinds of access (read, write) to each of
two different locations and two different data types (metadata, images) for a single access token 
using the fictitious `photo-api` type definition.

Also we could vary the numbers so that it is even more intuitive (2 each time currently).

Reply:
OK, these suggestions make sense. I'll expand the examples.